src/Atmos/Model/source.jl

```diff
@@ -49,11 +47,7 @@ function atmos_source!(
     t::Real,
     direction,
 )
-    if atmos.ref_state isa HydrostaticState
-        source.ρu -= (state.ρ - aux.ref_state.ρ) * aux.orientation.∇Φ
-    else
-        source.ρu -= state.ρ * aux.orientation.∇Φ
-    end
+    # Migrated to Σsources
```

Comment:
So is the idea that, once all other sources are converted (i.e. Coriolis, etc), these functions will become unnecessary? This function is left here for API compatibility?

Reply:
Yes, we have to leave it for now until all of the sources have been converted (`atmos_source!` sums over all source types and our source specification at the driver layer is not really changing).